workers/bubbleSort/run.py

```
@@ -0,0 +1,39 @@
+            message = json.loads(message)
+
+            if message["code"] == MSG_TYPE["REGISTER"]:
+                global IS_REGISTERED
+                IS_REGISTERED = True
+            else:
+                print(f"Task: {message['type']}, Value:{message['data']}")
+                resp = {
+                    "code": MSG_TYPE["RESPONSE"],
+                    "worker": True,
+                    "type": "bubbleSort",
+                    "value": bubble_sort(message["data"])
```

Ruff (0.8.2)

Undefined name `bubble_sort`

(F821)